x-pack/plugins/security_solution/public/management/pages/event_filters/view/components/flyout/index.test.tsx

```diff
@@ -70,7 +70,7 @@ describe('Event filter flyout', () => {
   it('should renders correctly', () => {
     const component = render();
     expect(component.getAllByText('Add Endpoint Event Filter')).not.toBeNull();
-    expect(component.getByText('cancel')).not.toBeNull();
+    expect(component.getByText('Cancel')).not.toBeNull();
```

Comment:
FYI: I'm always hesitant to use `getByText()` because it could break in the future if another element is introduced with the same text (probably not the case here). I normally prefer `getByTestId()`

Reply:
Yeah, that's totally true. I try to use getByTestId when more than one elements can have the same text, otherwise I prefer to use getByText as it not requires an extra prop in the component itself.